common/djangoapps/entitlements/helpers.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
```

Comment:
Could we define all of these methods on the model so that they can be used without having to import another file?

Reply:
Doing so would also allow us to simplify the names again.
`entitlement.is_expired` vs `is_entitlement_expired(entitlement)`

Reply:
This.

Reply:
:+1: